src/components/questions_answers/QnA.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Search from './QnA_Components/Search';
+// import exampleData from './exampleData';
+
+export default function QnA() {
+  return (
+    <section>
+      <h1>Questions &amp; Answers</h1>
+      <Search />
+      <div>
+        <p>Q: Would it fit super skinny people?</p>
+        <p>A: Probably Not</p>
+      </div>
+      <button type="submit">MORE ANSWERED QUESTIONS</button>
```

Comment:
Inside of the `<button>` tags here, I would recommend switching the `type` attribute from "submit" to "button" since the parent element of this is not a `<form>` element.